modules/lvms-deleting-lvmcluster-using-rhacm.adoc

```diff
@@ -0,0 +1,192 @@
+      type: ManagedClusterConditionAvailable
+  clusterSelector: <3>
+    matchExpressions:
+      - key: mykey
+        operator: In
+        values:
+          - myvalue
+----
+<1> The `spec.remediationAction` in `policy-template` is overridden by the preceding parameter value for `spec.remediationAction`.
+<2> This `namespace` field must have the `openshift-storage` value.
+<3> Configure the requirements to select the clusters. {lvms} is uninstalled on the clusters that match the selection criteria. 
+
+. Create the `Policy` CR by running the following command:
++
+[source,terminal]
+----
+$ oc create -f <file_name> -n <namespace>
+----
+
+. Create a `Policy` CR YAML file to check if the `LVMCluster` CR has been deleted:
++
+.Example `Policy` CR to check if the `LVMCluster` CR has been deleted
+[source,yaml]
+----
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
+  name: policy-lvmcluster-inform
+  annotations:
+    policy.open-cluster-management.io/standards: NIST SP 800-53
+    policy.open-cluster-management.io/categories: CM Configuration Management
+    policy.open-cluster-management.io/controls: CM-2 Baseline Configuration
+spec:
+  remediationAction: inform
+  disabled: false
+  policy-templates:
+    - objectDefinition:
+        apiVersion: policy.open-cluster-management.io/v1
+        kind: ConfigurationPolicy
+        metadata:
+          name: policy-lvmcluster-removal-inform
+        spec:
+          remediationAction: inform <1>
+          severity: low
+          object-templates:
+            - complianceType: mustnothave
+              objectDefinition:
+                kind: LVMCluster
+                apiVersion: lvm.topolvm.io/v1alpha1
+                metadata:
+                  name: my-lvmcluster
+                  namespace: openshift-storage <2>
+---
+apiVersion: policy.open-cluster-management.io/v1
+kind: PlacementBinding
+metadata:
+  name: binding-policy-lvmcluster-check
+placementRef:
+  apiGroup: apps.open-cluster-management.io
+  kind: PlacementRule
+  name: placement-policy-lvmcluster-check
+subjects:
+  - apiGroup: policy.open-cluster-management.io
+    kind: Policy
+    name: policy-lvmcluster-inform
+---
+apiVersion: apps.open-cluster-management.io/v1
+kind: PlacementRule
+metadata:
+  name: placement-policy-lvmcluster-check
+spec:
+  clusterConditions:
+    - status: "True"
+      type: ManagedClusterConditionAvailable
+  clusterSelector:
+    matchExpressions:
+      - key: mykey
+        operator: In
+        values:
+          - myvalue
+----
+<1> The `policy-template` `spec.remediationAction` is overridden by the preceding parameter value for `spec.remediationAction`.
+<2> The `namespace` field must have the `openshift-storage` value.
+
+. Create the `Policy` CR by running the following command:
++
+[source,terminal]
+----
+$ oc create -f <file_name> -n <namespace>
+----
+
+.Verification
+
+. Check the status of the `Policy` CRs by running the following command:
```

Comment:
```suggestion
* Check the status of the `Policy` CRs by running the following command:
```

Reply:
All your others are fine, but just this one needs to have a bullet instead of the single step https://redhat-documentation.github.io/modular-docs/#con-creating-procedure-modules_writing-mod-docs

Reply:
Updated as per the suggestion.